src/microbe_stage/Microbe.tscn

```diff
@@ -14,6 +14,7 @@ render_priority = 1
 shader = ExtResource( 4 )
 shader_param/wigglyNess = 1.0
 shader_param/movementWigglyNess = 1.0
+shader_param/dissolveValue = null
```

Comment:
Where did this come from? Is this change needed?

Reply:
This was set automatically by Godot

Reply:
This doesn't matter as another PR with this change was already merged (if I don't entirely misremember things).